.github/workflows/integration-tests.yml

```diff
@@ -94,7 +98,7 @@ jobs:
         env:
           SERVER_ADDRESS: localhost:8080
           INTEGRATION: aqueduct_demo
-        run: pytest . -rP --publish -n 5
+        run: pytest . -rP --publish #-n 5
```

Comment:
https://github.com/aqueducthq/aqueduct/runs/6761138874?check_suite_focus=true#step:16:36567 `-n 5` failed with a bunch of connection errors. I am not sure why this is particular to 310 tests.